cxx-sensors/src/main/java/org/sonar/cxx/sensors/utils/CxxReportSensor.java

```diff
@@ -252,7 +247,7 @@ private void saveViolation(SensorContext sensorContext, String ruleRepoKey,
     // handles file="" situation -- file level
     if ((filename != null) && (!filename.isEmpty())) {
       String root = sensorContext.fileSystem().baseDir().getAbsolutePath();
-      String normalPath = CxxUtils.normalizePathFull(filename, root);
+      String normalPath = CxxUtils.normalizePathFull(filename.replaceAll("\\\\", "/"), root);
```

Comment:
maybe we should use https://github.com/SonarSource/sonarqube/blob/master/sonar-plugin-api/src/main/java/org/sonar/api/utils/PathUtils.java?